dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowAction.java

```diff
@@ -112,43 +146,13 @@ public boolean isRequiresCheckout() {
     }
 
 	/**
-	 * @deprecated see {@link #setRequiresCheckoutOption(String)}
-	 * p-pp0()}
 	 * @param requiresCheckout
 	 */
 	@Deprecated
 	public void setRequiresCheckout(boolean requiresCheckout) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)